apps/condo/domains/settings/schema/MobileFeatureConfig.test.js

```diff
@@ -0,0 +1,195 @@
+/**
+ * Generated by `createschema settings.MobileFeatureConfig 'organization:Relationship:Organization:CASCADE; emergencyPhone:Text; commonPhone:Text; onlyProgressionMeterReadingsIsEnabled:Checkbox; meta:Json; ticketSubmittingIsEnabled:Checkbox'`
+ */
+
+const { makeLoggedInAdminClient, makeClient, UUID_RE, expectValuesOfCommonFields, catchErrorFrom } = require('@open-condo/keystone/test.utils')
+const {
+    expectToThrowAuthenticationErrorToObj, expectToThrowAuthenticationErrorToObjects,
+    expectToThrowAccessDeniedErrorToObj,
+} = require('@open-condo/keystone/test.utils')
+
+const { createTestOrganization, createTestOrganizationEmployee, createTestOrganizationEmployeeRole } = require('@condo/domains/organization/utils/testSchema')
+const { MobileFeatureConfig, createTestMobileFeatureConfig, updateTestMobileFeatureConfig } = require('@condo/domains/settings/utils/testSchema')
+const { makeClientWithNewRegisteredAndLoggedInUser } = require('@condo/domains/user/utils/testSchema')
+
+
+describe('MobileFeatureConfig', () => {
+    let admin, commonOrganization
+    beforeAll(async () => {
+        admin = await makeLoggedInAdminClient();
+        [commonOrganization] = await createTestOrganization(admin)
+
+    })
+    describe('CRUD tests', () => {
+        describe('accesses', () => {
+            describe('admin', () => {
```

Comment:
Tests for supports?)

Reply:
I don't see the point in this

Reply:
How do you know if something is broken?)

Reply:
there is no special logic. The support, like the admin, just has full access to this entity. There are tests for the admin. So the support is doing well

Reply:
If we decide in the future to restrict access for support users, the tests will not show any errors. For now, we have a separate type of user and have no test for them.